Joedmin-hw3/Joedmin-hw3/UsersNotFoundException.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+        
+namespace Joedmin_hw3
+{
+    public class UsersNotFoundException : Exception
+    {
+        public UsersNotFoundException(string message) : base(message)
```

Comment:
What user was not found? You can specify in the exception message instead of passing the full message.